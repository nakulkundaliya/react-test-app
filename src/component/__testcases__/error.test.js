import React from 'react';
import { shallow } from 'enzyme';
import Error from '../error';
let wrapped = shallow(<Error error="Not found" />);
describe('render error component', () => {
    it('should render the Error component correctly', () => {
        expect(wrapped).toMatchSnapshot();
    });
});