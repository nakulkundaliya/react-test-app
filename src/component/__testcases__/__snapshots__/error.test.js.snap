// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render error component should render the Error component correctly 1`] = `ShallowWrapper {}`;
